server/data/prisonRegisterApiClient.ts

```diff
@@ -0,0 +1,23 @@
+import config, { ApiConfig } from '../config'
+import RestClient from './restClient'
+import type { PrisonDto } from '../@types/prisonRegisterTypes'
+
+export default class PrisonRegisterApiClient {
+  restClient: RestClient
+
+  constructor(token: string) {
+    this.restClient = new RestClient('Prison register API', config.apis.prisonRegisterApi as ApiConfig, token)
+  }
+
+  async getPrisonDescription(agencyId: string): Promise<PrisonDto> {
+    return this.restClient.get({
+      path: `/prisons/id/${agencyId}`,
+    }) as Promise<PrisonDto>
+  }
+
+  async getPrisonOmuContactEmail(agencyId: string): Promise<string> {
+    return this.restClient.get({
+      path: `/secure/prisons/id/${agencyId}/offender-management-unit/email-address`,
+    }) as Promise<string>
+  }
```

Comment:
Also need some tests around this endpoint?

Reply:
This one not currently used, it was more 'get the basics in now' whilst I had all the info to hand - but yes, I'll add some as I'm pretty sure we will use it later.

Reply:
I've put some tests around this now.